clients/bigquery/bigquery.go

```diff
@@ -34,13 +34,12 @@ const (
 	describeNameCol               = "column_name"
 	describeTypeCol               = "data_type"
 	describeCommentCol            = "description"
-	// Storage Write API is limited to 10 MB, let's start out conservative and use 80% of that.
-	maxRequestByteSize = 10_000_000 * .8
+	// Storage Write API is limited to 10 MiB, subtract 50 KiB to account for request overhead.
+	maxRequestByteSize = (10 * 1024 * 1024) - (50 * 1024)
```

Comment:
is 50 kb fixed?

Reply:
After a bunch of testing the cutoff seemed to be around 30 KiB so I decided to go with 50 KiB to be safe.

Reply:
I tested it it with a pretty wide table so I think it should be fine, but if we do see errors we can increase it to 100 KiB.